pkg/artifactory/resource_artifactory_webhook.go

```diff
@@ -0,0 +1,375 @@
+package artifactory
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+
+	return false
+}
+
+var webhookTypesSupported = []string{
+	"artifact",
+	"artifact_property",
+	"docker",
+	"build",
+	"release_bundle",
+	"distribution",
+	"artifactory_release_bundle",
```

Comment:
artifactory_release_bundle and release_bundle  ... names looks very similar. 
Resource name "artifactory_artifactory_release_bundle_webhook" which has 2 artifactory mentioned ... looks so similar. Hope that is fine and causes no confusion for users.

Reply:
This is because they map directly to the type of webhook 'domain' the API support: https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-CreateaNewWebhookSubscription

Reply:
Also 'Artifactory Release Bundle' is a type of event in the UI

<img width="616" alt="Screen Shot 2022-02-14 at 10 03 11 AM" src="https://user-images.githubusercontent.com/27263/153920891-4c142876-347d-4786-af1a-c992c09932a0.png">